lectures/1-Introduction/LinearAlgebra.md

```diff
@@ -0,0 +1,240 @@
+
+```{contents} Contents
+:depth: 2
+```
+
+## Overview
+
+This lecture covers some basic linear algebra concepts and operations, that will serve as
+foundation for most of what follows. The goal is to provide some intuitive understanding of
+the concepts, without insisting on too much mathematical rigour. The most important goal is
+to introduce and define the concept of a tensor, without resorting to the usual mathematical
+definition, which is not very intuitive.
+
+Simultaneously, the lecture also showcases some of the features of
+[TensorKit.jl](https://github.com/Jutho/TensorKit.jl), a Julia package that is extremely
+well-suited for the demonstration of the concepts that are discussed.
+
+```{code-cell} julia
+using TensorKit
+```
+
+Before discussing tensor networks, it is necessary to understand what tensors are.
+Furthermore, before really understanding tensors, it is instructive to reiterate some basic
+concepts of linear algebra for the case of vectors and matrices, which are nothing but
+specific cases of tensors. In fact, many of the concepts and ideas that are introduced and
+discussed are defined in terms of thinking of tensors as vectors or matrices.
+
+In what follows, vectors and matrices will be thought of from the viewpoint of computers,
+where they are represented using regular one- and two-dimensional arrays of either real or
+complex numbers. Nevertheless, much of the discussion can be trivially generalized to
+arbitrary vector spaces and linear maps.
+
+## Vectors and Matrices
+
+In general, a vector is an object in a vector space, which can be described by a list of
+numbers that correspond to the components of the vector in some basis. For example, a vector
+in a two-dimensional space is in its most general form described by
+$\vec{v} = \left[v_1, v_2\right]^T$.
+
+As a reminder, the defining properties of vector spaces make sure that the following
+operations are well-defined:
+
+* Vectors can be added together, i.e. $\vec{v} + \vec{w}$ is a vector.
+* Vectors can be multiplied by scalars, i.e. $\alpha \vec{v}$ is a vector.
+* These operations behave as expected, i.e. there is a notion of associativity, commutativity, and distributivity.
+
+Given two such vector spaces (not necessarily distinct) it is possible to define a linear
+map between them, which is just a function that preserves the vector space structure. In
+other words, a linear map $A \colon W ← V$ maps vectors from one vector space $V$ to another
+vector space $W$. Because of the structure of vector spaces, and the requirement of
+linearity, such a map is completely determined by its action on the basis vectors of $V$.
+This leads in a very natural way to the notion of a matrix by considering the following
+construction, where $v_i$ are the basis vectors of $V$ and $w_i$ are the basis vectors of
+$W$:
+
+```{math}
+:label: eq:linear_map
+\begin{array}{rcl}
+A & : & W \leftarrow V\\
+  &   & v ↦ A(v) = \sum_j A_{ij} v_j = w_i
+\end{array}
+```
```

Comment:
This notation seems very confusing. The equation seems to interpret `v_j` as components of `v` in some basis, but before you say that `v_i`themselves are basis vectors.

Do you mean `v_i ↦ A(v_i) = \sum_j A_{ji} w_j`?